todo-app/.env.example

```
@@ -0,0 +1,65 @@
+LOG_LEVEL=debug
+
+DB_CONNECTION=sqlite
+# DB_HOST=127.0.0.1
+# DB_PORT=3306
+# DB_DATABASE=laravel
+# DB_USERNAME=root
+# DB_PASSWORD=
+
+SESSION_DRIVER=database
+SESSION_LIFETIME=120
+SESSION_ENCRYPT=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SESSION_ENCRYPT key should go before the SESSION_LIFETIME key

(UnorderedKey)